interface/super/edit_layout.php

```diff
@@ -1454,30 +1490,29 @@ function myChangeCheck() {
             $gmyname = $grparr[$group_id]['grp_title'];
 
             echo text($gdispname);
-
             // if not english and set to translate layout labels, then show the translation of group name
             if ($GLOBALS['translate_layout'] && $_SESSION['language_choice'] > 1) {
-                echo "<span class='translation'&gt;&gt;&gt;&nbsp; " . xlt($gdispname) . "</span>";
+                // echo "<span class='translation'&gt;&gt;&gt;&nbsp; " . xlt($gdispname) . "</span>";
+                echo "<span class='translation'>" . xlt($gdispname) . "</span>";
                 echo "&nbsp; ";
             }
-
             echo "&nbsp; ";
-            echo " <input type='button' class='btn btn-primary btn-sm addfield' id='addto~" . attr($group_id) . "' value='" . xla('Add Field') . "'/>";
+            echo " <input type='button' class='addfield' id='addto~$group_id' value='" . xla('Add Field') . "'/>";
             echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='btn btn-secondary btn-sm renamegroup' id='" . attr($group_id) . "~" . attr($gmyname) . "' value='" . xla('Rename Group') . "'/>";
-            /******************************************************************
+            echo " <input type='button' class='renamegroup' id='$group_id~$gmyname' value='" . xla('Rename Group') . "'/>";
             echo "&nbsp; &nbsp; ";
-            echo " <input type='button' class='deletegroup' id='" . attr($group_id) . "' value='" . xla('Delete Group') . "'/>";
-            ******************************************************************/
+            echo " <input type='button' class='deletegroup' id='$group_id' value='" . xl('Delete Group') . "'/>";
```

Comment:
the id variables should be run through attr() on the  code block. also need xla() above line and below line value labels